src/pages/HomePage/Language/Language.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { IoIosArrowDown, IoIosArrowUp } from 'react-icons/io';
+
+import type { Language } from 'types/languages';
+
+import { FrameworkComponent } from '../Framework/Framework';
+
+import styles from './Language.module.scss';
+
+interface LanguageProps {
+  language: Language;
+}
+
+export const LanguageComponent: React.FC<LanguageProps> = ({ language }) => {
+  const [showChildren, setShowChildren] = useState(false);
+  const changeShowChildrenState = () => {
+    setShowChildren(!showChildren);
+  };
+  return (
+    <>
+      <div
+        role="presentation"
```

Comment:
for interactive elements role 'presentation' is kind of counterintuitive, from accessibility standpoint probably a better option would be to use a button and style it to fit your needs